release-notes.md

```diff
@@ -1,5 +1,23 @@
 # Release notes
 
+## Aug 8, 2019
+
+**Places UI Update**
+
+#### New Features
+* Added a new list view showing POIs without the map
+* Added POI filtering options for city, state, country, and metadata
+* Auto create the first library in an org
+* Added POI sorting to the List View
+
+#### UI Updates
+* Moved list and details panel to right side of UI
+* Added a new search panel to the top of the UI
+* Autoselect libraries in POI create if only one library is present
+* Moved library management into a popup window
+* Added POI Count next to filters
```

Comment:
Please add a period at the end of the bullet points and reword some of the bullets, so that they read as follows:

- Moved the list and the details panel to right side of UI. --> Does "details" have an init cap?
- If only one library is present, this library is automatically selected when you create a POI.
- Added a POI count next to the filters.